sceptre/stack.py

```diff
@@ -146,14 +146,23 @@ class Stack:
     hooks = HookProperty("hooks")
 
     iam_role = create_deprecated_alias_property(
-        "iam_role", "sceptre_role", "4.0.0", "5.0.0"
+        "iam_role",
+        "sceptre_role",
+        deprecated_in="4.0.0",
+        removed_in=None,
```

Comment:
I assume that changing `removed_in` to None relax the requirement to remove this item in the next major release? 

What would be the message users see as a result of this change?